presto-main/src/test/java/com/facebook/presto/operator/scalar/TestMathFunctions.java

```diff
@@ -382,13 +382,58 @@ public void testRound()
     }
 
     @Test
+    public void testCut()
+    {
+        assertFunction("cut( 3)", BIGINT, 3);
+        assertFunction("cut(-3)", BIGINT, -3);
+        assertFunction("cut(CAST(NULL as BIGINT))", BIGINT, null);
```

Comment:
These null tests are not needed, since the engine will never call the function. Remove them all.